api/docs/v2/storefront/index.yaml

```
@@ -63,6 +63,16 @@ paths:
                     password_confirmation:
                       type: string
                       example: spree123
+                    public_metadata:
+                      type: object
+                      example:
+                        number_of_orders: 0
```

```suggestion
                        user_segment: 'supplier'
```

Let's use this example as number of orders is returned in `completed_orders` attribute